src/utils/token.js

```diff
@@ -0,0 +1,5 @@
+const AUTH_TOKEN = 'auth-token';
```

Comment:
This is the key/name that's being set in localStorage, correct? I think `AUTH_TOKEN` sounds like it's the actual token, I think you can just use the string directly in the localStorage calls, or rename this const to be clearer.